_topic_map.yml

```diff
@@ -124,6 +124,8 @@ Name: Installing
 Dir: installing
 Distros: openshift-origin,openshift-enterprise,openshift-webscale
 Topics:
+- Name: Understanding installations
```

Comment:
I think that for now, the duplication between this assembly and the one in the architecture book is fine, but we should consider changing the content in the architecture book later. I do think that this content makes more sense here, and I suggest opening a separate PR to change the prereqs for the installation assemblies to link to this new assembly. (I suggest a separate PR so that it would be easier to revert just that change in case someone disagees!)

Reply:
@kalexand-rh thank you for your review and for this suggestion. I have added those two action items to the Guided OpenShift 4 installation experience deliverables list.